modules/vuetify/index.js

```diff
@@ -0,0 +1,16 @@
+const path = require('path')
+
+module.exports = function nuxtVuetify (options) {
+  // Add assets
+  this.options.head.link.push({
+    rel: 'stylesheet',
+    type: 'text/css',
+    href: '//fonts.googleapis.com/css?family=Material+Icons'
```

Comment:
Some of their components use those icons (Radio buttons, Alerts, etc.)

What I could do though, is look if a `//fonts.googleapis.com/css?family=` already exists in the header and append `Material+Icons` to it. This would save a from doing a second request to Google Fonts.